.github/scripts/wg-onboard.sh

```
@@ -0,0 +1,750 @@
+log "Peer pool: peer1..peer${MAX_PEERS} (gap-fill order, create missing peers on demand)"
+
+CAN_CREATE_PEERS="false"
+if [[ ${#EXISTING_PEERS[@]} -gt 0 ]] \
+  || ssh_cmd bash -c "test -f $(remote_quote "$CONFIG_DIR/templates/peer.conf") || test -f $(remote_quote "$CONFIG_DIR/peer1/peer1.conf")" 2>/dev/null; then
+  CAN_CREATE_PEERS="true"
+else
+  die "No peer directories or templates under $CONFIG_DIR (cannot create missing peers)"
+fi
+
+peer_config_exists() {
+  local peer="$1" conf="$CONFIG_DIR/$peer/$peer.conf"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)